### PR TITLE
update migration api azure container and queue

### PR DESCRIPTION
**Pull request by JoanneHendrickson**
Category:  Content Fix

Description:  Added note regarding required endpoints if your tenant is hosted in GCC

### CODE REVIEW

File: docs/apis/migration-api-azure-container-and-queue.md [hunk 1]
```diff
@@ -63,3 +63,13 @@ The queue can be reused across multiple migration jobs so this call should not b
...
 Once those calls have been made, the rest of the flow remains the same for using the Migration API.
+
+### Required endpoints for goverment cloud
+
+If your tenant is hosted in a government cloud (GCC), you must have the proper endpoints set when calling the API. 
+*Example:*    usgovcloudapi.net 
+
+|**Required Endpoint**|**Why**|
+|:-----|:-----|
+|https://<spam><spam>*.blob.core.usgovcloudapi.<spam><spam>net|Migration API Azure Government requirement|
```
Comment (andrewconnell) [inline docs/apis/migration-api-azure-container-and-queue.md:74]: Invalid link... please correct before submitting.

File: docs/apis/migration-api-azure-container-and-queue.md [hunk 2]
```diff
@@ -63,3 +63,13 @@ The queue can be reused across multiple migration jobs so this call should not b
...
+|https://<spam><spam>*.queue.core.usgovcloudapi.<spam><spam>net|Migration API Azure Government requirement|
```
Comment (andrewconnell) [inline docs/apis/migration-api-azure-container-and-queue.md:75]: Invalid link... please correct before submitting.

Comment (JoanneHendrickson) [reply docs/apis/migration-api-azure-container-and-queue.md:75]: Hi Andrew -- that link was deliberately padded with <spam> characters so the link would not resolve -- we want to have it appear simply as an example

Comment (andrewconnell) [reply docs/apis/migration-api-azure-container-and-queue.md:75]: I see... can we use something other than "spam"? The technique you're leveraging should be using "span", not "spam", as the former is a real HTML tag that won't get rendered or flagged.

Alternatives could be to use code references. Check this gist, flip to RAW mode to see the source. Using a backtick will make it code:
https://gist.github.com/andrewconnell/a25748d8efab81035dca488b16bc0166

**Comment by JoanneHendrickson** [comment]
Thanks Andrew

**Pull request merged**